step-api/src/main/java/org/jenkinsci/plugins/workflow/steps/AbstractSynchronousNonBlockingStepExecution.java

```diff
@@ -61,21 +78,4 @@ private static synchronized ExecutorService getExecutorService() {
         return executorService;
     }
 
-    private static class StepRunner implements Runnable {
```

Comment:
There's no accounting for taste... but I'll try to follow your preferred notation in the future in order to have a uniform style across the project.